howto/inference.md

```diff
@@ -53,3 +53,26 @@ python generate.py --quantize gptq.int4
 
 For instance, `falcon-40b` "only" requires ~24 GB to generate using this technique, otherwise more than +40GB are required. On an A100 it takes 1 token/sec.
 However, ~32 GB were required during the conversion step.
+
+
+## Run a large model on multiple consumer devices
+
+You can also use the Fully-Sharded Data Parallel (FSDP) distributed strategy to leverage multiple devices to perform inference. This will allow you to run models that wouldn't fit in a single card by sharding them across several.
+
+For instance, `falcon-40b` would require ~80 GB of GPU memory to run on a single device. We can instead run it on 4 A100 40GB GPUs:
```

Comment:
A100 aren't exactly consumer hardware. I think it's fine as long as we mention some consumer card in the section below where you run on 2 devices.

Reply:
Done